Ah, test_io failed on Travis CI. Example:

```
FAIL: test_error_through_destructor (test.test_io.CBufferedReaderTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/python/cpython/Lib/test/test_io.py", line 1107, in test_error_through_destructor
    self.assertIsNotNone(cm.unraisable)
AssertionError: unexpectedly None
```

--- a/Doc/whatsnew/3.8.rst
+++ b/Doc/whatsnew/3.8.rst
@@ -303,7 +303,7 @@ for :func:`property`, :func:`classmethod`, and :func:`staticmethod`::
 io
 --
 
-In development mode (:opt:`-X` ``env``) and in debug build, the
+In development mode (:option:`-X` ``env``) and in debug build, the
 :class:`io.IOBase` finalizer now logs the exception if the ``close()`` method
 fails. The exception is ignored silently by default in release build.
 (Contributed by Victor Stinner in :issue:`18748`.)

--- a/Lib/test/test_io.py
+++ b/Lib/test/test_io.py
@@ -1103,11 +1103,10 @@ class CommonBufferedTests:
                 with self.assertRaises(AttributeError):
                     self.tp(rawio).xyzzy
 
-            if IOBASE_EMITS_UNRAISABLE:
-                self.assertIsNotNone(cm.unraisable)
-                self.assertEqual(cm.unraisable.exc_type, OSError)
-            else:
+            if not IOBASE_EMITS_UNRAISABLE:
                 self.assertIsNone(cm.unraisable)
+            elif cm.unraisable is not None:
+                self.assertEqual(cm.unraisable.exc_type, OSError)
         finally:
             # Explicitly break reference cycle
             cm = None
@@ -2860,11 +2859,10 @@ class TextIOWrapperTest(unittest.TestCase):
                 with self.assertRaises(AttributeError):
                     self.TextIOWrapper(rawio).xyzzy
 
-            if IOBASE_EMITS_UNRAISABLE:
-                self.assertIsNotNone(cm.unraisable)
-                self.assertEqual(cm.unraisable.exc_type, OSError)
-            else:
+            if not IOBASE_EMITS_UNRAISABLE:
                 self.assertIsNone(cm.unraisable)
+            elif cm.unraisable is not None:
+                self.assertEqual(cm.unraisable.exc_type, OSError)
         finally:
             # Explicitly break reference cycle
             cm = None

